sdk/src/main/java/com/fingerprint/api/FingerprintApi.java

```diff
@@ -380,18 +380,96 @@ public ApiResponse<VisitorsGetResponse> getVisitsWithHttpInfo(String visitorId,
+    private Long start;
+    private Long end;
+    private Boolean reverse;
+    private Boolean suspect;
+
+    public String getPaginationKey() {
+      return paginationKey;
+    }
+
+    public SearchEventsFilter setPaginationKey(String paginationKey) {
+      this.paginationKey = paginationKey;
+      return this;
+    }
+    public String getVisitorId() {
+      return visitorId;
+    }
+
+    public SearchEventsFilter setVisitorId(String visitorId) {
+      this.visitorId = visitorId;
+      return this;
+    }
+    public String getBot() {
+      return bot;
+    }
+
+    public SearchEventsFilter setBot(String bot) {
+      this.bot = bot;
+      return this;
+    }
+    public String getIpAddress() {
+      return ipAddress;
+    }
+
+    public SearchEventsFilter setIpAddress(String ipAddress) {
+      this.ipAddress = ipAddress;
+      return this;
+    }
+    public String getLinkedId() {
+      return linkedId;
+    }
+
+    public SearchEventsFilter setLinkedId(String linkedId) {
+      this.linkedId = linkedId;
+      return this;
+    }
+    public Long getStart() {
+      return start;
+    }
+
+    public SearchEventsFilter setStart(Long start) {
+      this.start = start;
+      return this;
+    }
+    public Long getEnd() {
+      return end;
+    }
+
+    public SearchEventsFilter setEnd(Long end) {
+      this.end = end;
+      return this;
+    }
+    public Boolean getReverse() {
+      return reverse;
+    }
+
+    public SearchEventsFilter setReverse(Boolean reverse) {
+      this.reverse = reverse;
+      return this;
+    }
+    public Boolean getSuspect() {
+      return suspect;
+    }
+
+    public SearchEventsFilter setSuspect(Boolean suspect) {
+      this.suspect = suspect;
+      return this;
+    }
+  }
   /**
    * Get events via search
    * Search for identification events, including Smart Signals, using multiple filtering criteria. If you don't provide `start` or `end` parameters, the default search range is the last 7 days.  Please note that events include mobile signals (e.g. `rootApps`) even if the request originated from a non-mobile platform. We recommend you **ignore** mobile signals for such requests. 
    * @param limit Limit the number of events returned.  (required)
-   * @param visitorId Unique [visitor identifier](https://dev.fingerprint.com/reference/get-function#visitorid) issued by Fingerprint Pro. Filter for events matching this `visitor_id`.  (optional)
-   * @param bot Filter events by the bot detection result, specifically:    `all` - events where any kind of bot was detected.   `good` - events where a good bot was detected.   `bad` - events where a bad bot was detected.   `none` - events where no bot was detected.  (optional)
-   * @param ipAddress Filter events by IP address range. The range can be as specific as a single IP (/32 for IPv4 or /128 for IPv6)  All ip_address filters must use CIDR notation, for example, 10.0.0.0/24, 192.168.0.1/32  (optional)
```

Comment:
Are we losing these detailed comments from Intellisense, or are they still present somehow?

Reply:
You are right. I [added code comments](https://github.com/fingerprintjs/fingerprint-pro-server-api-java-sdk/pull/100/commits/bb04c67facc2602088d4cbfbb86558d564faf33a) for getters and setters in filter object

Reply:
Nice, thanks!